etc/adminhtml/system.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <tab id="gloo" translate="label" sortOrder="10">
```

Comment:
I think you should create your own tab, so that other sso extensions that use this as base can easily have their own configuration sub tabs if they wish to. Secondly the tab with id gloo is created by an extension gloo base that is not open source. Create your own tab.